raiden-dapp/.env.v2

```diff
@@ -0,0 +1,6 @@
+VUE_APP_PUBLIC_PATH=/v2/
+VUE_APP_ALLOW_MAINNET=true
+VUE_APP_SERVICE_WORKER_DISABLED=true
+VUE_APP_IMPRINT=https://raiden.network/privacy.html
+VUE_APP_TERMS=https://github.com/raiden-network/light-client/blob/master/TERMS.md
+VUE_APP_CONFIGURATION_URL=config.production.json
```

Comment:
That one need to point to `/v2/config.production.json` I think. I did that [here](https://github.com/raiden-network/light-client/blob/436a9530451fed1ddb816b3f2b5a30aee76a0af9/raiden-dapp/.env.production#L2) and I'm pretty certain that was necessary because we make a raw `fetch` call [here](https://github.com/raiden-network/light-client/blob/e8d0f259d6c20763e43186ef8ce49d81c6ef2d25/raiden-dapp/src/services/config-provider.ts#L35). So we should not load the configuration of Arbitrum One.

Reply:
This is a relative path. Shouldn't it download from BASE_URL?

Reply:
I tested here by hand, and indeed, both `config.production.json` and `./config.production.json` download from root.
I fixed it now.

Reply:
👍🏾